esp/src/eclwatch/GridDetailsWidget.js

```diff
@@ -58,6 +60,17 @@ define([
             this.inherited(arguments);
             this.initGrid();
             this.initContextMenu();
+
+            if (!this.displayOpenButton) {
+                var openButtonElements = [
+                    document.querySelector(`[widgetid="${this.id}Open"]`),
+                    document.querySelector(`#${this.id}ContainerNode`),
+                    document.querySelector(`#${this.id}ContainerNode`).nextElementSibling
+                ];
+                for (var i = 0; i < openButtonElements.length; i++) {
+                    openButtonElements[i].style.display = "none";
+                }
+            }
```

Comment:
I think the other examples I alluded to use this:  `dojo.destroy(this.id + "Open");`
`

Reply:
Btw, I wasn't sure if you meant to replace that entire loop etc with just the dojo.destroy(...). Those other two items are siblings rather than children of the open btn and weren't being removed:

![image](https://user-images.githubusercontent.com/79224539/128745151-e088f688-75b2-415a-92c1-590877ca3445.png)

Reply:
If they don't have IDs already it would be cleaner to simply add them and then delete similar to the open?  See:  eclwatch\templates\GridDetailsWidget.html (they do have IDs)